src/__tests__/StocksList.test.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render, cleanup } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import fetchMock from 'jest-fetch-mock';
+import StocksList from '../components/stocks/StocksList';
+
+import MockData from '../__mocks__/MockStocks';
+import store from '../redux/configureStore';
+import { fetchData } from '../redux/stokes/stokes';
+
+fetchMock.enableMocks();
+
+let StocksListProvider;
+beforeEach(() => {
+  fetch.resetMocks();
+  fetch.mockResponseOnce(JSON.stringify(MockData));
+  store.dispatch(fetchData());
+  StocksListProvider = render(
+    <Provider store={store}>
+      <Router>
+        <StocksList />
+      </Router>
+    </Provider>,
+  );
+});
+
+afterEach(() => {
+  cleanup();
+});
+
+describe('StocksList component', () => {
+  it('test if apple inc. is there', async () => {
+    const apple = await StocksListProvider.findByText('Apple Inc.');
+    expect(apple).toBeTruthy();
+  });
+
+  //   it('Renders the reports component', async () => {
+  //     const reports = await StocksListProvider.findByText(
+  //       'Daily Metrics Summary'
+  //     );
+  //     expect(reports).toBeTruthy();
+
+  //     const topCompanies = await StocksListProvider.findAllByRole('feed');
+  //     expect(topCompanies).toHaveLength(3);
+  //   });
+
+  //   it('Renders the cards for company listings', async () => {
+  //     const cards = await StocksListProvider.findAllByRole('img');
+  //     expect(cards).toHaveLength(30);
+  //   });
+
+  //   it('Renders the search bar', async () => {
+  //     const input = await StocksListProvider.findByRole('search');
+  //     expect(input).toBeInTheDocument();
+  //   });
```

Comment:
- [ ] I think it would be better to delete commented code because it makes your codebase unclean.

Reply:
> * [x]  I think it would be better to delete commented code because it makes your codebase unclean.

Sure, I forgot to remove that. 


> * [x]  Please create a good README with project title, description, live demo link, screenshots to make it more informative for users. Also make sure that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

This is Done too.

> * [x]  I recommend following the given design [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Currently, your app does not have the board on top (I marked it with blue color below in the screenshot) 👇

Done as well.

> * [ ]  Please record a video for your project. Below are the guidelines 👇
>   ![Screenshot (236)](https://user-images.githubusercontent.com/67087664/158946815-8b9ccf27-4e19-4ab5-8f13-839581cab666.png)

Working on this now, Thanks @elyor-doniyorov

Reply:
All Done 
And here is the link to the video demo : https://www.loom.com/share/5b84d35272b741578f9bf63f0388680b

Reply:
next reviewer, please note that the live demo will not work as expected until I merge the testing branch to dev branch